rubocop.yml

```diff
@@ -388,6 +393,8 @@ Style/MethodCallWithArgsParentheses:
     - to_not
     - eq
     - be
+    - puts
+    - yield
```

Comment:
I think these should follow the rule. Do we have many of them to warrant an exception?

Reply:
In `prawn-dev` there are 7 `yield`s and 2 `puts`. In `prawn` there are 70 `yield`s and 11 `puts`.

Reply:
Thanks for the stats. Let's keep `yield` here but make `puts` follow the rule.

Ideally I'd like `yield` follow the rule, too, but it looks like a lot of work.

Reply:
Done! Sorry it took so long!